<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SaveOtherDialog</class>
 <widget class="QDialog" name="SaveOtherDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>541</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Save Other</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Directory</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="directory_lineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="browse_pushButton">
       <property name="text">
        <string>Browse to Directory</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Filename</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="filename_lineEdit"/>
     </item>
    </layout>
   </item>
   <item row="2" column="0" rowspan="2">
    <layout class="QGridLayout" name="workspace_list_layout"/>
   </item>
   <item row="2" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="save_button">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancel_button">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="1">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Save Formats</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="RKH_checkBox">
       <property name="text">
        <string>RKH</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="NxCanSAS_checkBox">
       <property name="text">
        <string>NxCanSAS</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="CanSAS_checkBox">
       <property name="text">
        <string>CanSAS</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
